examples/gkurve/atlas.zig

```diff
@@ -0,0 +1,327 @@
+//! This implementation comes from https://gist.github.com/mitchellh/0c023dbd381c42e145b5da8d58b1487f
```

Comment:
Can you add [the LICENSE file](https://gist.githubusercontent.com/mitchellh/0c023dbd381c42e145b5da8d58b1487f/raw/15d618acc3ba084e58a154fa44bc0b181a73a73c/LICENSE.md) to this folder, too? Name it `LICENSE.atlas`

Reply:
Dp I move all the doc comment to the LICENSE.atlas or just the first line?